Runtime/Model/BacktraceStackFrame.cs

```diff
@@ -1,31 +1,50 @@
 ﻿using Backtrace.Unity.Json;
+using Backtrace.Unity.Types;
 using System;
+using System.Collections.Generic;
 using System.Diagnostics;
+using System.IO;
 using System.Reflection;
 
 namespace Backtrace.Unity.Model
 {
     public class BacktraceStackFrame
     {
+
+        private static string[] _frameSeparators = new string[] { "::", ":", "." };
+
         /// <summary>
         /// Function where exception occurs
         /// </summary>
         public string FunctionName;
 
+        internal BacktraceStackFrameType StackFrameType = BacktraceStackFrameType.Unknown;
+
         /// <summary>
-        /// Line number in source code where exception occurs
+        /// Function file name
         /// </summary>
-        public int Line;
+        public string FileName
+        {
+            get
+            {
+                return string.IsNullOrEmpty(Library)
+                        ? GetFileNameFromFunctionName()
+                        : Library.IndexOfAny(Path.GetInvalidPathChars()) == -1 && Path.HasExtension(Path.GetFileName(Library))
+                            ? Path.GetFileName(Library).Trim()
+                            : GetFileNameFromFunctionName();
+            }
+        }
+
 
         /// <summary>
-        /// IL Offset
+        /// Line number in source code where exception occurs
         /// </summary>
-        public int? Il;
```

Comment:
We added this variable when we started developing Backtrace-csharp integration and wanted to use it for getting line numbers when symbols are available. Later when we started thinking about symbols solution, we thoughts we should use different name. 

Because we still working on symbols solution for dotnet apps and we send this info twice, I decided there is no reason to do that. Users don't see this info in UI. Backend expect iloffset in different variable, and il json variable is useless. This change remove it from json object and save few allocations